runtime/doc/repeat.txt

```diff
@@ -135,7 +135,10 @@ q			Stops recording.
 
 							*@*
 @{0-9a-z".=*+}		Execute the contents of register {0-9a-z".=*+} [count]
-			times.  Note that register '%' (name of the current
+			times. In visual mode, repeat the previous @{0-9a-z":*} for
+			each selected line.
```

Comment:
Should these be referenced in `:h default-mappings` and have a `v_@-default` tag instead?

Reply:
The "previous" also seems wrong here.

Reply:
> Should this be referenced in `:h default-mappings` and have a `v_@-default` tag instead? (And similarly for the others)

Agree, although:

1.  the `v_@-default` part, would be adding the `desc` into the keymap set and adding the tag in the `repeat.txt`, right? (just in case I'm missing something else).
2. how about adding the `v_@-default` tag to the @ section? it would be like the following:
```
							*@* *v_@-default*
@{0-9a-z".=*+}		Execute the contents of register {0-9a-z".=*+} [count]
			times. In Visual mode, repeat the contents of register
			{0-9a-z".=*+} for each selected line.
```

> The "previous" also seems wrong here.

Yes, it's wrong, also going to fix it.

Reply:
1. Yup. That `desc` looks good, thanks!

2. Personally I'd keep it consistent with how other defaults are described.
  Usually the default mapping is mentioned in its own paragraph after the related command, and ends with a link to `|default-mappings|`.

For example, I think the following below the section for `@@` could cover both `@{0-9a-z".=*+}` and `@@`, while remaining relatively clear and short:

```vimdoc
							*v_@-default*
{Visual}@{0-9a-z".=*+}	In Visual mode, execute the contents of the register
{Visual}@@		for each selected line. See |visual-repeat|.
			|default-mappings|
```

...though Justin may have his own opinion on this. :innocent: